package.json

```diff
@@ -92,7 +92,8 @@
     "fuse.js": "^7.0.0",
     "lexical": "^0.19.0",
     "micromatch": "^4.0.8",
-    "polished": "^4.1.3"
+    "polished": "^4.1.3",
+    "react-hook-form": "^7.53.2"
```

Comment:
How come did you decide to install the library?

I think you can get away with just typing the hookformError as
```
{
    type: string
    message?: string
  }
```